From 9e9fed10fd5d5036b41bf10eccd771308bc135c8 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:51 -0400
Subject: [usb] pl2303: rename pl2303_encode_baud_rate

Message-id: <1410887463-20674-580-git-send-email-dzickus@redhat.com>
Patchwork-id: 95576
O-Subject: [RHEL7 PATCH 579/591] USB: pl2303: rename pl2303_encode_baud_rate
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 79816824c1ae73542d9523283ce353142b67563e
Author: Johan Hovold <jhovold@gmail.com>
Date:   Sun Dec 29 19:23:10 2013 +0100

    USB: pl2303: rename pl2303_encode_baud_rate

    Rename baud-rate encoding function to match tty naming.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 43848b9..df001a2 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -313,7 +313,7 @@ static int pl2303_set_control_lines(struct usb_serial_port *port, u8 value)
  return retval;
 }
 
-static void pl2303_encode_baudrate(struct tty_struct *tty,
+static void pl2303_encode_baud_rate(struct tty_struct *tty,
      struct usb_serial_port *port,
      u8 buf[4])
 {
@@ -337,7 +337,7 @@ static void pl2303_encode_baudrate(struct tty_struct *tty,
  if (!baud)
   return;
 
- /* Set baudrate to nearest supported value */
+ /* Set baud rate to nearest supported value */
  for (i = 0; i < ARRAY_SIZE(baud_sup); ++i) {
   if (baud_sup[i] > baud)
    break;
@@ -465,7 +465,7 @@ static void pl2303_set_termios(struct tty_struct *tty,
  }
 
  /* For reference buf[0]:buf[3] baud rate value */
- pl2303_encode_baudrate(tty, port, &buf[0]);
+ pl2303_encode_baud_rate(tty, port, &buf[0]);
 
  /* For reference buf[4]=0 is 1 stop bits */
  /* For reference buf[4]=1 is 1.5 stop bits */
-- 
1.7.1